torchgeo/datasets/oscd.py

```diff
@@ -0,0 +1,302 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""OSCD dataset."""
+
+import glob
+import os
+from typing import Callable, Dict, List, Optional, Sequence, Union
+
+import matplotlib.pyplot as plt
+import numpy as np
+import rasterio
+import torch
+from matplotlib.figure import Figure
+from PIL import Image
+from torch import Tensor
+
+from ..datasets.utils import draw_semantic_segmentation_masks
+from .geo import VisionDataset
+from .utils import download_url, extract_archive, sort_sentinel2_bands
+
+
+class OSCD(VisionDataset):
+    """OSCD dataset.
+
+    The Onera Satellite Change Detection dataset addresses the issue of detecting
+    changes between satellite images from different dates. Imagery comes from
+    Sentinel-2 which contains varying resolutions per band.
+
+    Dataset format:
+
+    * images are 13-channel tifs
+    * masks are single-channel pngs where no change = 0, change = 255
+
+    Dataset classes:
+
+    0. no change
+    1. change
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * https://doi.org/10.1109/IGARSS.2018.8518015
```

Comment:
Can you add:
```rst
.. versionadded:: 0.2
```
I forgot to do this for other new features that have been added, but we want to document when new features are added: 85768b81732e763a0ec742ae0f7a2f7c5aa44000

Reply:
Where should this go?

Reply:
I'll put it under the link to the paper like in the other datasets

Reply:
> Where should this go?

See 85768b81732e763a0ec742ae0f7a2f7c5aa44000 for examples